Formula/podman.rb

```diff
@@ -7,6 +7,18 @@ class Podman < Formula
     url "https://github.com/containers/podman/archive/v4.2.0.tar.gz"
     sha256 "15f8bc59025ccd97dc9212a552e7274dfb79e1633b02d6a2a7f63d747eadb2f4"
 
+    patch do
+      url "https://github.com/containers/common/commit/030b7518103cfd7b930b54744d4a4510b659fdc2.patch?full_index=1"
+      sha256 "7c00abe7728d6438abcdb69ce6efa43503dcbb93bcb2d737f6ca4aa553e2eeb5"
+      directory "vendor/github.com/containers/common"
+    end
+
+    patch do
+      url "https://github.com/containers/common/commit/4e6828877b0067557b435ec8810bda7f5cb48a4f.patch?full_index=1"
+      sha256 "01c4c67159f83f636a7b3a6007a010b1336c83246e4fb8c34b67be32fd6c2206"
+      directory "vendor/github.com/containers/common"
+    end
```

Comment:
Can we add a comment above the patches explaining what they're for and when they can be removed?

Reply:
Done.

I expect the next upgrade will break the build if the patches are still enabled since they'll clearly conflict with the lines already there, so it will be easy to tell that it's time to drop them.

Thanks!